### PR TITLE
Fix bug with enum closure parameter

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: json_serializable/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## 0.5.5
+## 0.5.4+1
 
 * Added named `formatOutput` parameter to `jsonPartBuilder`.
```
Comment (natebosch) [inline json_serializable/CHANGELOG.md]: adding a parameter (A feature someone can depend on) should be done in a minor release, not a bug fix release. The previous version in this file was better.

File: json_serializable/lib/src/type_helpers/enum_helper.dart [hunk 2]
```diff
@@ -34,6 +34,11 @@ class EnumHelper extends TypeHelper {
     var wrappedExpression =
         simpleExpression.hasMatch(expression) ? expression : '{$expression}';
 
+    var closureArg = consts.closureArg;
```
Comment (natebosch) [inline json_serializable/lib/src/type_helpers/enum_helper.dart]: so was the import unused before? How is it that you could add a prefix without changing any other lines?

Comment (kevmoo) [reply json_serializable/lib/src/type_helpers/enum_helper.dart]: It wasn't unused before. I'm creating a local var that maps exactly to the field `closureArg`. This way the code that does the generation doesn't change at all...

File: json_serializable/CHANGELOG.md (hunk 1 above)
Comment (kevmoo) [reply json_serializable/CHANGELOG.md]: Er, uh. Yup. Will fix.

...although the named arg will be useful to very few people. But fair...

Comment (kevmoo) [reply json_serializable/CHANGELOG.md]: Went the other way and removed the changelog entry – jsonPartBuilder is not public!

**Pull request merged**